config.m4

```diff
@@ -365,11 +365,11 @@ PHP_ARG_WITH(libmongoc, whether to use system libmongoc,
   if test "$PHP_LIBMONGOC" != "no"; then
     AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
     AC_MSG_CHECKING(for libmongoc)
-    if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists libmongoc-1.0 && $PKG_CONFIG --exists libmongoc-priv; then
+    if test -x "$PKG_CONFIG" && $PKG_CONFIG --exists libmongoc-1.0; then
       if $PKG_CONFIG libmongoc-1.0 --atleast-version 1.3.3; then
-        LIBMONGOC_INC=`$PKG_CONFIG libmongoc-priv --cflags`
```

Comment:
Wouldn't `--atleast-version 1.3.3` need to be changed to `1.4.0`?

Are there similar changes required for the libbson package dependency? I expect they need to be kept in sync.